### PR TITLE
Contact form of the portfolio

### PR DESCRIPTION
**Pull request by DanUshindi**
In that pair Programing my partner and me:

-Edited my contact form 
-Added all the requirements 
-Made change on the CSS to style well contact form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -179,53 +183,53 @@ <h1 class="project-name">Base Apparel coming soon page</h1>
...
-          <textarea class="textarea" name="comment">  Enter text here</textarea>
-       </form>
-       <div class="fnl-btn faa">
-        <button class="btn" disabled>See project</button>
-      </div>
+          I’d love to help with it
+        </p>
+        <form action="https://formspree.io/f/mgedkakg" method="POST" class="prjct">
+          <input class="wrt" maxlength="30" type="text" placeholder="Enter your name">
+          <input class="wrt" type="Email" placeholder="Enter your mail">
```
Comment (nicupop729) [inline index.html:200]: It is not the best to receive emails after users submit the form, only to find out that the form is completed anonymously.

- [ ] Kindly make sure that you require as mandatory that the input fields are completed.

**Review by nicupop729 — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️

Hi @DanUshindi, 

Very good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



 ## To Highlight

- [ ] Very organised and clean code :heavy_check_mark: 
- [ ] Linters are passing :heavy_check_mark: 

### Required Changes ♻️

- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Here we have the [5 basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to follow in Microverse projects.
- [ ] We can choose more appropriate names for our branches. In this case, it could represent an action like "build-contact-form"; later on, if we want to bring a change, we could name it "refactor-contact-form".
- [ ] Branch names should start with a small letter, and commit messages should start with a capital letter.
- [ ] Use descriptive commit messages - each commit should be small enough to describe it in one sentence. The commit message should be written in the imperative tense and capitalized.
- [ ] The commit messages should be descriptive, representing the changes brought to the code. Your commit messages are repetitive and are not descriptive. 

**Please make sure to follow all these rules and I kindly ask you to read again these [5 basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -179,53 +183,53 @@ <h1 class="project-name">Base Apparel coming soon page</h1>
...
-        <button class="btn" disabled>See project</button>
-      </div>
+          I’d love to help with it
+        </p>
+        <form action="https://formspree.io/f/mgedkakg" method="POST" class="prjct">
+          <input class="wrt" maxlength="30" type="text" placeholder="Enter your name" required>
+          <input class="wrt" type="Email" placeholder="Enter your mail" required>
+          <textarea class="textarea" maxlength="500"  name="comment"
+            placeholder="Enter your text"></textarea>
+          <button class="btn" type="submit">Submit</button>
```
Comment (mateo951) [inline index.html:203]: Addressed ✔ Form is working as expected 🙌

File: index.html [hunk 3]
```diff
@@ -179,53 +183,53 @@ <h1 class="project-name">Base Apparel coming soon page</h1>
...
-          <textarea class="textarea" name="comment">  Enter text here</textarea>
-       </form>
-       <div class="fnl-btn faa">
-        <button class="btn" disabled>See project</button>
-      </div>
+          I’d love to help with it
+        </p>
+        <form action="https://formspree.io/f/mgedkakg" method="POST" class="prjct">
+          <input class="wrt" maxlength="30" type="text" placeholder="Enter your name" required>
+          <input class="wrt" type="Email" placeholder="Enter your mail" required>
```
Comment (mateo951) [inline index.html:200]: Addressed 😎

**Review by mateo951 — APPROVED** [review]
### Hi Team,

Well done implementing the changes requested by your previous reviewer 🔥
Your project is complete!  There is nothing else to say other than... it's time to merge it ✔ 
### Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**